### PR TITLE
Move account deletion from a long press to a context menu.

### PR DESCRIPTION
**Pull request by twaddington**
**What's this do?**
Move account deletion from a long press to a context menu. The account list layouts have been simplified to accommodate the new context menu. Additionally, some classes have been renamed for better consistency.

**Screenshots**
- [Screenshot_1605917069](https://user-images.githubusercontent.com/107117/100018348-2ccbe580-2d91-11eb-91c4-297ad0eeba48.png)
- [Screenshot_1605917073](https://user-images.githubusercontent.com/107117/100018342-2b9ab880-2d91-11eb-9d74-f181532f1276.png)
- [Screenshot_1605917081](https://user-images.githubusercontent.com/107117/100018352-2d647c00-2d91-11eb-8460-4deda019a34f.png)

**Why are we doing this? (w/ JIRA link if applicable)**
https://jira.nypl.org/browse/SIMPLY-2864

**How should this be tested? / Do these changes have associated tests?**
Try deleting an added account by going to "Settings -> Accounts" and tapping on the context menu. This should work the same as before except the long press is no longer needed.

**Dependencies for merging? Releasing to production?**
n/a

**Has the application documentation been updated for these changes?**
n/a

**Did someone actually run this code to verify it works?**
@twaddington ran the vanilla app.

### CODE REVIEW

**Review by io7m — APPROVED** [review]
I like it!

I actually approved this hours ago but apparently Github didn't register it...

EDIT: No I didn't, this is a different PR...

File: simplified-ui-accounts/src/main/java/org/nypl/simplified/ui/accounts/AccountListFragment.kt [hunk 1]
```diff
@@ -34,14 +34,14 @@ import org.nypl.simplified.ui.thread.api.UIThreadServiceType
  * A fragment that shows the set of accounts in the current profile.
  */
 
-class AccountsFragment : Fragment() {
+class AccountListFragment : Fragment() {
```
Comment (io7m) [inline simplified-ui-accounts/src/main/java/org/nypl/simplified/ui/accounts/AccountListFragment.kt:37]: :+1:

**Comment by io7m** [comment]
Ok, I _actually_ approve this now. :laughing: 

Really glad to see this. I've always hated long presses.

**Comment by twaddington** [comment]
Thanks for the review Mark!

**Pull request merged**